File: rest/authorized-connect-apps/list-get-example-1/list-get-example-1.5.x.php
<?php
// Get the PHP helper library from twilio.com/docs/php/install
require_once '/path/to/vendor/autoload.php'; // Loads the library
use \Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";
$client = new Client($sid, $token);

// Loop over the list of authorized_apps and echo a property for each one
foreach ($client->account->authorizedConnectApps->read() as $authorizedApp) {
    echo $authorizedApp->connectAppHomepageUrl;
}

File: rest/authorized-connect-apps/list-get-example-1/list-get-example-1.5.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Loop over authorized_apps and print out a property for each one
@client.authorized_connect_apps.list
       .each do |authorized_app|
         puts authorized_app.connect_app_homepage_url
       end
